src/__tests__/scheduling/appointments/edit/EditAppointment.test.tsx

```diff
@@ -94,32 +94,28 @@ describe('Edit Appointment', () => {
   })
 
   it('should render an edit appointment form', async () => {
-    const { wrapper } = await setup()
+    const { wrapper } = await setup(expectedAppointment, expectedPatient)
 
-    expect(wrapper.find(AppointmentDetailForm)).toHaveLength(1)
+    expect(wrapper.find(Spinner)).toHaveLength(1)
```

Comment:
I am not sure i understand how finding the Spinner confirms the presence of the AppointmentDetailForm

Reply:
Whoops yeah, it really doesn't.